lib/membrane_rtc_engine/endpoints/webrtc/rtp_munger.ex

```diff
@@ -93,50 +97,44 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPMunger do
       when buffer.metadata.rtp.is_padding? == true do
     # This function clause deals with padding packets
 
-    metadata =
-      buffer.metadata
-      |> put_in(
-        [:rtp, :sequence_number],
-        calculate_seq_num(rtp_munger.highest_incoming_seq_num + 1, rtp_munger)
-      )
-      # Timestamps can be duplicated, so we always take the highest one we seen to avoid causing any ordering problems
-      |> put_in([:rtp, :timestamp], rtp_munger.last_timestamp)
+    if rtp_munger.last_marker == true do
```

Comment:
I didn't add comment but let's leave a short note that we need to send padding packets only at frame boundary